plugins/SolarSystemEditor/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 
-SET(SOLARSYSTEMEDITOR_MAJOR "0")
```

Comment:
Why 1.1.0? This definitely is 0.2.8, in better case 0.3.0, but not 1.1.0

Reply:
Is it even meaningful to keep separate plugin version numbers? The plugins are all static. Just set to the main program version. If not, I agree, this mini change does not warrant a big version number change.

Reply:
Syncing plugin version with version of main application is acceptable for me. The separate plugin version numbers is coming by historical reasons and we have no guarantee that in future we'll not produce dynamical plugins again.

Reply:
Sure. We could then split out the numbers again if that appears useful.

Reply:
IMHO we should get rid of plugin version info _for now_. I think it is a smokescreen, a source of confusion and triggers needless discussions and draws effort to keep it up-to-date. 

But for what?

I don't ever see _any_ of the current plugins _ever_ appear outside of Stellarium. If that ever happens, then the version can be shown again. **So I propose to hide the version info and freeze the numbers for now.**

The only advantage could be to draw the attention to the user as to what happened. Assuming the user understands SemVer...

About 1.0: the plugin is **no longer in alpha/beta phase**. It is used extensively by the end users. And 1.1.0 signifies a minor update w.r.t. the previous version. 

x= major/important/breaking change
y = minor change / improvement
z = bug fix

If we would keep 0.y.z, how else can new plugins be marked as being in alpha/beta phase? Only new plugins deserve the 0.y.z numbering, until they are released/part of a Stellarium release. Unless it is uncertain how the plugin behaves, and _end users_ can play with it, knowing it is still in _beta_.

This is mainly a matter of **courage**. Stop thinking that everything should be 0.y until everything works fine, because that will **never** happen. 

1.0 = released.

Reply:
@axd1967 if you something do not known then this is not meaning that this is not exists!